# Medizinische Genomanalysen 2019
Material and information about the lecture "Medizinische Genomanalysen" 2019

## Overview
* Fork & clone the repository of each assignment
* Each student needs to work on his/her assignment (no team work)
* When done, commit and push the results to your github repository
* All assignments need to be completed by **07.05.2019**

## Data and packages for Python assignments
#### Data
* BAM file: ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/HG00096/alignment/HG00096.chrom11.ILLUMINA.bwa.GBR.low_coverage.20120522.bam 
* VCF file:
ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/analysis/IonTorrent_TVC_03162015/AmpliseqExome.20141120.NA24385.vcf
* VCF file: 
ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/analysis/IonTorrent_TVC_03162015/AmpliseqExome.20141120.NA24143.vcf
* VCF file:
ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/analysis/IonTorrent_TVC_03162015/AmpliseqExome.20141120.NA24149.vcf
  
#### Packages
* Biopython (http://biopython.org/)
* pybedtools (https://daler.github.io/pybedtools/)
* NCBI (http://biopython.org/DIST/docs/api/Bio.Entrez-module.html)
* pybam (https://github.com/JohnLonginotto/pybam)
* pysam (http://pysam.readthedocs.io/en/latest/usage.html)
* cyvcf2 (https://github.com/brentp/cyvcf2)
* pyvcf (http://pyvcf.readthedocs.io/en/latest/)
* HGVS - does only work with Python2! (https://hgvs.readthedocs.io/en/master/)


## Assignment 1
Repository: [medizinische_genomanalysen_2019_assignment_1](https://github.com/spabinger/medizinische_genomanalysen_2019_assignment_1)

## Assignment 2
Repository: [medizinische_genomanalysen_2019_assignment_2](https://github.com/spabinger/medizinische_genomanalysen_2019_assignment_2)

## Assignment 3
Repository: [medizinische_genomanalysen_2019_assignment_3](https://github.com/spabinger/medizinische_genomanalysen_2019_assignment_3)



## Student projects

### Exercise - Clinical sequencing

#### Your TODOs
1. Per group present the selected software
   * Software: 10min 
2. *Optional: Perform 6 analysis steps (if possible) using FASTQ files*
3. *Optional: Present the performed analysis*


#### OPTIONAL: Perform the following steps 
1. *Optional: Data Import – take FASTQ files from:*
   * *Illumina:  ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data*
   * *Ion Torrent: (Only for Ion Reporter)  ftp://ftp-trace.ncbi.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/ion_exome/*
   * *Requirement: human, genome (no RNASeq, methylation, …)*
2. *Optional: Quality Control & QC Report*
3. *Optional: Mapping*
4. *Optional: Variant Calling (SNV, Indel)*
5. *Optional: Variant Annotation*
6. *Optional: Interpretation & Decision Making*

#### Student groups 
1. Illumina basespace  --  https://basespace.illumina.com/home/index 
   * xx
   * xx
   * xx
2. Ion Reporter  --  https://ionreporter.lifetechnologies.com/ir/ 
   * xx 
   * xx
   * xx
3. Galaxy  --  https://main.g2.bx.psu.edu/ 
   * xx
   * xx
   * xx
4. Chipster  --  http://chipster.csc.fi/ 
   * xx
   * xx
   * xx
5. GenePattern  --  https://genepattern.broadinstitute.org/gp
   * xx
   * xx
   * xx










